client/src/ui/organisms/placement/index.scss

```diff
@@ -144,6 +144,18 @@ section.place {
   }
 }
 
+section.place.hp-main {
+  background-color: var(--place-hp-main-background);
+  display: flex;
+  justify-content: center;
```

Comment:
Should these two be set on the child `a` tag instead, as well as `width: 100%;` on both this and that, so that users can click on the background as well? I just tried to do it on staging, and was mildly annoyed I actually had to click on the image.